### PR TITLE
add google meet incident report (Oct 25 2021)

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
ping @icco

File: README.md [hunk 1]
```diff
@@ -209,6 +209,8 @@
 
 [Google](https://gist.github.com/jomo/2bae3821acb433d0446d). A mail system emailed people more than 20 times. This happened because mail was sent with a batch cron job that sent mail to everyone who was marked as waiting for mail. This was a non-atomic operation and the batch job didn't mark people as not waiting until all messages were sent.
 
+[Google Meet](https://www.google.com/appsstatus/dashboard/incidents/k71P8nHp32hgcMSsC3mR). The Google Meet Livestream feature experienced disruptions that caused intermittent degraded quality of experience for a small subset of viewers, starting 25 October 2021 0400 PT and ending 26 October 2021 1000 PT. Quality was degraded for a total duration of 4 hours (3 hours on 25 October and 1 hour on 26 October). During this time, no more than 15% of livestream viewers experienced higher rebuffer rates and latency in livestream video playback. We sincerely apologize for the disruption that may have affected your business-critical events. We have identified the cause of the issue and have taken steps to improve our service.
```
Comment (icco) [inline README.md:212]: Please replace Google Meet with just Google.

**Review by icco — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (chenrui333) [reply README.md:212]: Sounds good, let me update it now.

**Review by icco — APPROVED** [review]

**Comment by icco** [comment]
Thanks!

**Comment by chenrui333** [comment]
My pleasure, thanks for the merge! :)

**Pull request merged**